### DISCUSSION TITLE
how can I change the date style

### DISCUSSION THREAD

**Discussion by xtccc** [hugo-sid/hugo-blog-awesome#112] [Q&A] [answered]
I want to change the date format ，can you add a config to setting the date format?

**Comment by hugo-sid** [comment, marked as answer]
Hi @xtccc, I have added support for setting custom date format. You can take a look at the following code, which illustrates how to set the format in the config file. 

https://github.com/hugo-sid/hugo-blog-awesome/blob/e0abd993c32bc7381fc4942d6565baa1bc6ba087/exampleSite/config.toml#L69-L70

Please update your theme to use this feature.

Thanks for using HBA (Hugo Blog Awesome). Don't forget to star the repo, if you liked this theme.
